Add README (#36)
QW3RT2U10PU/hpi-gamejam2025: README.md

> # This game was made wiht Godot 4.5.1 for a private 48h GameJam.

Reviewer: ```suggestion
# Vampire-Vlad's Vegan Ventures

This game was made with Godot 4.5.1 for a private 48h game jam.
```

> Vampire Vlad's Vegan Ventures is a platformer game where a young vampire wants to visit Berlin and stop eating humans. On his journey, he will encounter a strong enemy...

Reviewer: ```suggestion
Vampire-Vlad's Vegan Ventures is a platformer game where a young vampire wants to visit Berlin and stop eating humans. On their journey, they will encounter a strong enemy...
```
